### PR TITLE
ClusterLoader - Wait for pods update

### PR DESCRIPTION
**Pull request by krzysied**

### CODE REVIEW

**Comment by krzysied** [comment]
/hold 
Requires #271 
Possible conflict with #272

File: clusterloader2/pkg/measurement/common/wait_for_pods.go [hunk 1]
```diff
@@ -70,31 +70,39 @@ func (*waitForRunningPodsMeasurement) Execute(config *measurement.MeasurementCon
...
-		}
-		if runningPodsCount == desiredPodCount {
-			return nil
+			if log {
+				glog.Infof("WaitForRunningPods: namespace(%s), labelSelector(%s), fieldSelector(%s): running %d / %d", namespace, labelSelector, fieldSelector, runningPodsCount, desiredPodCount)
+			}
+			if runningPodsCount == desiredPodCount {
+				return nil
+			}
+			time.Sleep(defaultWaitForPodsInterval)
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/wait_for_pods.go]: select {
case <- stopCh
case <- time.After()
}

[we want to break as soon as we can]

**Pull request merged**